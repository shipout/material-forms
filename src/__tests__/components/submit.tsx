import React from 'react'
import ButtonSubmit from '../../components/submit'
import { shallow } from 'enzyme'
import * as enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

enzyme.configure({ adapter: new Adapter() })
describe('ButtonSubmit', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<ButtonSubmit>TEST</ButtonSubmit>)
    expect(component).toMatchSnapshot()
  })
  /*
  it('should render correctly in "debug" mode', () => {
    const component = shallow(
      <ButtonSubmit
        handleClick={() => {
          console.log('asdsad')
        }}
      >
        TEST
      </ButtonSubmit>
    )
    expect(component).toMatchSnapshot()

    component.find('input').simulate('click')
    //mockedButton.find('.Button__btn').simulate('click');
    //mockedButton.find('button.Button__btn').simulate('click');
  })
  */
})
