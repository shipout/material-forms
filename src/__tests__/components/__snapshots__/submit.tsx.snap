// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonSubmit should render correctly in "debug" mode 1`] = `
<WithStyles(ForwardRef(Button))
  className="button 
         
         
        "
>
  TEST
</WithStyles(ForwardRef(Button))>
`;

exports[`ButtonSubmit should render correctly in "debug" mode 2`] = `
<WithStyles(ForwardRef(Button))
  className="button 
         
         
        "
  onClick={[Function]}
>
  TEST
</WithStyles(ForwardRef(Button))>
`;
